### PR TITLE
Ambassadors blog

### PR DESCRIPTION
**Pull request by anatoliykmetyuk**
~Can be merged only after #1614.~ done

### CODE REVIEW

**Comment by SethTisue** [comment]
I'll take a look tomorrow (Wednesday).

**Review by dagmendez — APPROVED** [review]

**Comment by SethTisue** [comment]
Darja and I will sit with this tomorrow.

File: blog/_posts/2024-03-26-ambassadors-initiative.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* promote a positive image of Scala
+* recognize and support the independent work of active community members
+
+## What are Ambassadors? 🤔
+
+Ambassadors are leaders and key figures in the Scala community.
+
+Ambassadors have volunteered to work with the Scala Center and the community to:
+
+* help newcomers learn Scala and become part of the community
```
Comment (anatoliykmetyuk) [inline blog/_posts/2024-03-26-ambassadors-initiative.md:23]:
```suggestion
* Help newcomers learn Scala and become part of the community
```

File: blog/_posts/2024-03-26-ambassadors-initiative.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* improve communication between everyone involved with Scala
```
Comment (anatoliykmetyuk) [inline blog/_posts/2024-03-26-ambassadors-initiative.md:24]:
```suggestion
* Improve communication between everyone involved with Scala
```

File: blog/_posts/2024-03-26-ambassadors-initiative.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+* aid and support each other's efforts
```
Comment (anatoliykmetyuk) [inline blog/_posts/2024-03-26-ambassadors-initiative.md:25]:
```suggestion
* Aid and support each other's efforts
```

**Review by anatoliykmetyuk — COMMENTED** [review]
@SethTisue your changes LGTM, added a minor suggestion for consistency with other bullet points.

**Review by SethTisue — APPROVED** [review]
also approved just now by Darja, for posting later today — we’re just waiting until we can get a few more names on the ambassador page, then let's merge.

**Comment by SethTisue** [comment]
live at https://scala-lang.org/blog/2024/03/28/ambassadors-initiative.html

**Pull request merged**